src/adapters/EasyRtcAdapter.js

```diff
@@ -31,16 +31,17 @@ class EasyRtcAdapter extends NoOpAdapter {
     this.easyrtc.joinRoom(roomName, null);
   }
 
-  // options: { datachannel: bool, audio: bool }
+  // options: { datachannel: bool, audio: bool, video: bool }
   setWebRtcOptions(options) {
     // this.easyrtc.enableDebug(true);
     this.easyrtc.enableDataChannels(options.datachannel);
 
-    this.easyrtc.enableVideo(false);
+    this.easyrtc.enableVideo(options.video);
     this.easyrtc.enableAudio(options.audio);
```

Comment:
If we don't use enableVideo and enabledAudio here, we may still connect without the mic if permission was denied? see #55

Reply:
no, that on the _connect method of the adapter, i have not changed that behavior will be the same as before with audio, if denied will not connect. 

Original: https://github.com/networked-aframe/networked-aframe/blob/master/src/adapters/EasyRtcAdapter.js#L219
New: https://github.com/networked-aframe/networked-aframe/blob/db099d8396113b83f9cd256b1a4e7115c175fd45/src/adapters/EasyRtcAdapter.js#L260